docs/azure-config.md

```diff
@@ -160,7 +143,7 @@ In the root of your Ark directory, run:
   ```
```

Comment:
Just wondering why this `apply` is on the entire directory, and in the other examples we specify each file.

Reply:
I don't even remember the specific reason, but there was some difference in either the contents of the azure dir vs. the others, or the order in which things needed to be applied.  I know that for AWS at least, we have 2 different versions of the Ark deployment in the examples/ dir, so we wouldn't want to apply the whole directory.

Reply:
Probably could be made to be consistent with the others, though.

Reply:
I'll add a docs issue for this.